scripts/emailings/migrate_files_path.php

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env php
```

Comment:
A lot of user does not have access to the command line (most mutualized hosting and cloud services)
This is why such features are centralized into the page 
/install/repair.php

Can you move the call of the function into the repair.php page ?